downscale/estimate_climo_anom_regression.f90

```diff
@@ -467,7 +467,7 @@ end subroutine heapsort
     do i = 1, nstns, 1
       y (i) = prcp_data (i, t)
       y_tmean (i) = (tair_data(1, i, t)+tair_data(2, i, t)) / 2.0d0
-      y_trange (i) = abs (tair_data(2, i, t)-tair_data(1, i, t))
+      y_trange (i) = tair_data(2, i, t)-tair_data(1, i, t)
```

Comment:
The abs() was included in the case that Tmax is larger than Tmin in the input files.  We can remove it but that may allow for negative DTR...

Reply:
We should really add a warning to the user and then stop the program if Tmax > Tmin.